offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/search/SearchAffiliationViewModel.groovy

```diff
@@ -0,0 +1,56 @@
+package life.qbic.portal.offermanager.components.affiliation.search
+
+import groovy.beans.Bindable
+import life.qbic.datamodel.dtos.business.Affiliation
+import life.qbic.portal.offermanager.dataresources.persons.AffiliationResourcesService
+
+/**
+ * <h1>ViewModel for SearchAffiliationView</h1>
+ *
+ * <p>Determines state of the view. Listens to data events.</p>
+ *
+ * @since 1.0.0
+ */
+class SearchAffiliationViewModel {
+
+    /**
+     * A list of available affiliations. All items are of class {@link Affiliation}
+     */
+    ObservableList affiliations
+    @Bindable Affiliation selectedAffiliation
```

Comment:
maybe an optional?

Reply:
nice idea 👍 I will implement it as an Optional<Affiliation>